From cca6711ae5053162a1cd59da2db7824359f627a6 Mon Sep 17 00:00:00 2001
From: Liu Xiaowen <b37945@freescale.com>
Date: Thu, 27 Oct 2011 16:31:17 +0800
Subject: [PATCH 197/635] ENGR00160838 fix 1080p playback rotation 180 with
 noise.

change the overlay output format to uyvy from rgbp.

Signed-off-by: Liu Xiaowen <b37945@freescale.com>
---
 mx5x/hwcomposer/FG_device.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mx5x/hwcomposer/FG_device.cpp b/mx5x/hwcomposer/FG_device.cpp
index 8b4ab5b..4e2ed49 100755
--- a/mx5x/hwcomposer/FG_device.cpp
+++ b/mx5x/hwcomposer/FG_device.cpp
@@ -226,7 +226,7 @@ HWCOMPOSER_LOG_RUNTIME("-------------FG_device::init()---open fb0-------------")
    // m_top = 0;
     m_width = def_info.xres;//info.xres;
     m_height = def_info.yres;//info.yres;
-    m_format = fourcc('R', 'G', 'B', 'P');//('U', 'Y', 'V', 'Y');
+    m_format = fourcc('U', 'Y', 'V', 'Y');
 
   	info.reserved[0] = def_info.reserved[0];
   	info.reserved[1] = def_info.reserved[0];
-- 
1.8.0

